test/Common.h

```diff
@@ -44,6 +44,7 @@ struct ConfigException : public util::Exception {};
 struct CommonOptions: boost::noncopyable
 {
 	boost::filesystem::path evmonePath;
```

Comment:
Does it matter which one is the evmone path?

Reply:
I left `evmonePath` here to ensure the old behaviour of defining this via `--evmonepath`. `evmonePath` may also be defined automatically, if an evmone library was found in default paths.

Internally `vmPaths` is always used. If no shared library is defined via `--vm`, `evmonePath` will always be added to the `vmPaths`. See [`CommonOptions::parse(..)`](https://github.com/ethereum/solidity/blob/583bbbb5a463dfe1e9d8504883b9c53a9a9977b0/test/Common.cpp#L144).

But this also means, that in the situation where just one library was defined with`--vm`, the evmone library need to be defined via `--vm` as well.

Reply:
I think it would be better to do it properly. This is a test runtime, we can make breaking changes any time.

Reply:
I removed now `--evmonepath`. Multple evmc vms can now be only defined with `--vm`. If no vms where defined with `--vm`, it will search for an `evmone` library automatically. if additionally `--ewasm` was defined, it will automatically search for `hera` too.